src/plugins/auto/auto_executable_network.cpp

```diff
@@ -92,7 +92,7 @@ IE::Parameter AutoExecutableNetwork::GetMetric(const std::string& name) const {
                 bThroughputEnabledInPlugin =
                     _autoSContext->_core->GetConfig(deviceInfo.deviceName,
                         CONFIG_KEY(PERFORMANCE_HINT)).as<std::string>() == CONFIG_VALUE(THROUGHPUT);
-            } catch (...) {
+            } catch (const IE::Exception&) {
```

Comment:
IE::Exception is deprecated class, why not use ov::Exception?

Reply:
For currently, plugin still use legacy IE::Exception, but for api level it is better to ov::Exception.
Sorry to mislead.